client/shaders/volumetric_light/opengl_fragment.glsl

```diff
@@ -58,7 +58,7 @@ vec3 getDirectLightScatteringAtGround(vec3 v_LightDirection)
 	// for Nitrogen at 532nm (green), 2e25 molecules/m3 in atmosphere
 	const vec3 beta_r0_l = vec3(3.3362176e-01, 8.75378289198826e-01, 1.95342379700656) * beta_r0; // wavelength-dependent scattering
 
-	const float atmosphere_height = 15000.; // height of the atmosphere in meters
+	const float atmosphere_height = 50000.; // height of the atmosphere in meters
```

Comment:
Why this change? I think 15000 came from some paper.
I guess the troposphere extends to ~15km, whereas the stratosphere extends to 50km :)

In any any, why this number change?

Reply:
This is pretty much just aesthetic preference I suppose, with how big minetest‘s sun is, you would intuitively expect its light to tint when it’s still higher in the sky than you would in real life… Wether that justifies changing it is a good question. Maybe this could be added to settings or something instead? I do quite prefer how it looks with the increased height personally :/